.travis.yml

```diff
@@ -5,6 +5,8 @@ node_js:
   - "6.17"
   - "8.16"
   - "10.16"
+  - "12.16"
+  - "13.12"
```

Comment:
Thanks! Can you remove the odd-numbers? We don't test those here.

Reply:
sure thanks